engine/tests/ElggCoreGetEntitiesTest.php

```diff
@@ -348,6 +348,40 @@ public function testElggAPIGettersTSPValidTypeValidSubtype() {
 	}
 
 	/**
+	 * Valid type, ELGG_ENTITIES_NO_VALUE as subtype
+	 */
+	public function testElggAPIGettersTSPValidTypeNoValueSubtype() {
```

Comment:
Add a docblock that describes the test. That makes it easier to improve the name later.